### PR TITLE
Unused backend variables

### PR DESCRIPTION
**Pull request by cybojenix**

### CODE REVIEW

File: govwifi-backend/variables.tf [hunk 1]
```diff
@@ -34,36 +30,10 @@ variable "bastion-server-ip" {}
...
-
-variable "bastion-set-cronjobs" {}
-
-variable "ssh-key-name" {}
-
-variable "backend-instance-count" {}
-
-variable "backend-min-size" {}
-
-variable "backend-cpualarm-count" {}
```
Comment (emileswarts) [inline govwifi-backend/variables.tf]: Just did a quick search and it looks like this variable is still in use in
terraform/modules/govwifi-backend/modules/ecs-autoscaling/scaling-policy.tf

File: govwifi-backend/variables.tf [hunk 2]
```diff
@@ -34,36 +30,10 @@ variable "bastion-server-ip" {}
...
-
-variable "backend-elb-count" {}
-
-variable "aws-account-id" {}
```
Comment (emileswarts) [inline govwifi-backend/variables.tf]: Same with this one:
terraform/modules/govwifi-backend/iam-roles.tf

File: govwifi-backend/variables.tf (hunk 1 above)
Comment (cybojenix) [reply govwifi-backend/variables.tf]: looks like that module isn't used anymore. will remove it

File: govwifi-backend/variables.tf (hunk 2 above)
Comment (cybojenix) [reply govwifi-backend/variables.tf]: realised this afterwards, it's moved to https://github.com/alphagov/govwifi-terraform/pull/173/files#diff-5766a9533fb6c3b8a8e3c3cc2bd9e8bdR9 with the other aws vars

**Pull request merged**